lib/cli/command-processor.js

```diff
@@ -88,12 +88,18 @@ module.exports = class CommandProcessor extends EventEmitter {
   }
 
   call(interfaceName, methodName, args, callback) {
-    if (!this.cli.client) return callback(new Error('Not connected'));
```

Comment:
@lundibundi
What do you say about removing redundant indentation by leaving line 91 as it was, and just adding this:
```js
if (!this.cli.connection) return callback(new Error('Connection in progress'));
```